packages/mobile/src/features/auth/components/SignIn/LogIn.tsx

```diff
@@ -55,33 +55,23 @@ const ForgotOpacity = styled.TouchableOpacity({
   });
   
 export interface LogInProps {
-    // label: string;
-    // formikProps: any;
-    // formikKey: string;
     navigation: any;
   }
 
-  
- 
-
 class LogIn extends React.Component<LogInProps> {
     loginSubmit = async (value: any, action: any) => {
         console.log(value);
-
-        const dataResponse = await Axios.post('/auth/signin', {
-            "email": `${value.email}`,
-            "password": `${value.password}`
-             })
-             console.log(dataResponse);
-
-            const token = dataResponse.data.token;
-            console.log(token);
-            AsyncStorage.setItem('token', token);
+        // const dataResponse = await Axios.post('/auth/signin', {
```

Comment:
co to?

Reply:
jak w opisie, za komentowałam dlatego że jest to potrzebny kod ale póki co nie zapisujemy nowych użytkowników do bazy, a chciałam przekierować od razu na widok listy